ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -50,6 +50,7 @@ export enum TransactionAuthenticatorVariant {
   TransactionAuthenticatorEd25519 = 0,
   TransactionAuthenticatorMultiEd25519 = 1,
   TransactionAuthenticatorMultiAgent = 2,
+  TransactionAuthenticatorSecp256k1 = 3,
   TransactionAuthenticatorFeePayer = 4,
```

Comment:
```suggestion
  TransactionAuthenticatorFeePayer = 4,
  TransactionAuthenticatorSecp256k1 = 5,
```

This will always match the Rust enum- so if in doubt for these variants check the *.rs file we link and make sure it matches